content/ar/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,59 @@
+---
+title: مرحبًا، مينيكيوب
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "ابدأ الآن"
+    weight: 10
+    post: >
+       <p>مستعد للبدء؟ قم ببناء عنقود كوبرناتيز بسيط لتشغيل عينة تطبيق .</p>
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+يوفر هذا الدليل كيفية تشغيل تطبيق عينة على كوبرناتيز باستخدام مينيكيوب.
+يقدم الدليل صورة حاوية تستخدم نجينكس لإعادة توجيه جميع الطلبات.
+
+## {{% heading "objectives" %}}
+
+* نشر تطبيق عينة على مينيكيوب.
+* تشغيل التطبيق.
+* عرض سجلات التطبيق.
+
+## {{% heading "prerequisites" %}}
+
+يفترض هذا البرنامج التعليمي أنك قد قمت بإعداد `مينيكيوب` بالفعل.
+يرجى الرجوع إلى __الخطوة 1__ في [بدء مينيكيوب](https://minikube.sigs.k8s.io/docs/start/) للحصول على تعليمات التثبيت.
+
+{{< note >}}
+قم بتنفيذ التعليمات فقط في __الخطوة 1، التثبيت__. الباقي مغطى في هذه الصفحة.
+{{< /note >}}
+
+كما يتعين عليك تثبيت `kubectl`.
+يرجى الرجوع إلى [تثبيت الأدوات](/docs/tasks/tools/#kubectl) للحصول على تعليمات التثبيت.
+
+<!-- lessoncontent -->
+
+## إنشاء عنقود مينيكيوب
+
+```shell
+minikube start
+```
+
+## فتح  لوحة المعلومات
+
+افتح لوحة معلومات كوبرناتيس. يمكنك فعل ذلك بطريقتين مختلفتين:
+
+{{< tabs name="dashboard" >}}
+{{% tab name="Launch a browser" %}}
+افتح **سطر أوامر* جديدًا، وقم بتشغيل:
+```shell
+# ابدأ سطر أوامر جديد واترك هذا الأمر يعمل.
+minikube dashboard
+```
+
+الآن، قم بالتبديل إلى سطر الأوامر حيث قمت بتشغيل `minikube start`.
```

Comment:
I think there is remaining content in the English doc.

PTAL: after `## Open the Dashboard` in https://raw.githubusercontent.com/kubernetes/website/dev-1.29-ar.1/content/en/docs/tutorials/hello-minikube.md

```
{{< note >}}
The `dashboard` command enables the dashboard add-on and opens the proxy in the default web browser.
You can create Kubernetes resources on the dashboard such as Deployment and Service.

To find out how to avoid directly invoking the browser from the terminal and get a URL for the web dashboard, see the "URL copy and paste" tab.

By default, the dashboard is only accessible from within the internal Kubernetes virtual network.
The `dashboard` command creates a temporary proxy to make the dashboard accessible from outside the Kubernetes virtual network.

To stop the proxy, run `Ctrl+C` to exit the process.
After the command exits, the dashboard remains running in the Kubernetes cluster.
You can run the `dashboard` command again to create another proxy to access the dashboard.
{{< /note >}}


....

```